nodangle.gap

```diff
@@ -5,6 +5,7 @@ import "Extensions/probabilities.hh"
 import "Extensions/shapes.hh"
 import "Extensions/mea.hh"
 import "Extensions/probing.hh"
+import "Extensions/outside.hh"
```

Comment:
could you rename this file into `cofold_nodangle.gap`? We would otherwise break other parts for "normal" folding